internal/docker/image.go

```diff
@@ -19,3 +19,19 @@ func ToImgNameAndTag(nt reference.NamedTagged) ImgNameAndTag {
 		Tag:  nt.Tag(),
 	}
 }
+
+func NormalizeRefName(ref string) (string, error) {
+	named, err := reference.ParseNormalizedNamed(ref)
+	if err != nil {
+		return "", err
+	}
+	return named.String(), nil
+}
+
+func MustNormalizeRefName(ref string) string {
```

Comment:
This could live in the test package as it's only used there

Reply:
that makes sense, though i'm inclined to leave it near the function it `Must`'s? 🤷‍♀️